.travis.yml

```diff
@@ -4,7 +4,6 @@ cache: pip
 
 env:
   - TOXENV=py27
-  - TOXENV=py33
```

Comment:
travis-ci no longer seems to support 33

Reply:
Yep, it is failing also on my test env :)